### ISSUE TITLE
rule: commands should be in uppercase 

### ISSUE THREAD

**Issue by dgutson** [hadolint/hadolint#955]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
hadolint should check that commands are in uppercase (check https://github.com/rcjsuen/dockerfile-utils/blob/master/src/dockerValidator.ts#L1254)

### Actual behavior
No message if `from` is in lowercase

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 2.12.0
```
